src/framework/audioplugins/internal/registeraudiopluginsscenario.cpp

```diff
@@ -49,6 +49,23 @@ void RegisterAudioPluginsScenario::init()
     }
 }
 
+io::paths_t RegisterAudioPluginsScenario::newPluginPaths()
```

Comment:
It's better to reuse this function in registerNewPlugins() to avoid code duplication. It should also be const